reflex/constants/base.py

```diff
@@ -19,36 +19,34 @@ class Dirs(SimpleNamespace):
     # The frontend directories in a project.
     # The web folder where the NextJS app is compiled to.
     WEB = ".web"
+    # The web folder where the NextJS app is exported to.
+    WEB_EXPORT = ".web_export"
```

Comment:
do we still need this then?

Reply:
I meant to remove this, thanks for catching it 👍